inspirehep/utils/record_getter.py

```diff
@@ -55,9 +55,9 @@ def wrapper(*args):
         try:
             return f(*args)
         except Exception as e:
-            current_app.logger.error('Error in retrieving record {0} '
-                                     'caused by {1}'.format(repr(args),
-                                                            repr(e)))
+            current_app.logger.error(
+                'Error in retrieving record %s caused by %s',
```

Comment:
I would do something like `'Error in retrieving record %s caused by {}'.format(repr(e))` so that we have a different event for each cause of not finding an recid.